public/app/plugins/datasource/influxdb/datasource.ts

```diff
@@ -176,6 +176,28 @@ export default class InfluxDatasource extends DataSourceApi<InfluxQuery, InfluxO
     return false;
   }
 
+  interpolateVariablesInQueries(queries: InfluxQuery[]): InfluxQuery[] {
+    let expandedQueries = queries;
+    if (queries && queries.length > 0) {
+      expandedQueries = queries.map(query => {
+        const expandedTags = query.tags.map(tag => {
```

Comment:
think this needs to check for raw queries as well (that only have the text field)

Reply:
If it's a rawQuery then there is no 100& guarantee that tags property exists (could be undefined)